### PR TITLE
Don't be too shallow

### PR DESCRIPTION
**Pull request by doublek**
Don't git unshallow the repo that semaphore has checkout fully anyway.

### CODE REVIEW

**Comment by marvin-tigera** [comment]
OK, I will merge the pull request when it's ready, leave the commits as is when I merge it, and leave the branch after I've merged it.

**Review by Brian-McM — APPROVED** [review]

**Pull request merged**
